ui/uihandler.go

```diff
@@ -920,3 +922,14 @@ func setHostAndPort(inputPort string, addr *net.TCPAddr) {
 		urlHostPort = "localhost" + portStr
 	}
 }
+
+// httpToHTTPS replaces "http://" in url with "https://".
+func httpToHTTPS(url string) (parsedURL string) {
+	if strings.HasPrefix(url, "http://") {
```

Comment:
no, as we discussed, this should be

addHTTPS() {
  if doesn't start with https://; add https://
}

as before I could do `istio.fortio.io:443` in the destination box and check grpc-secure and it worked and should continue to work